changelog/@unreleased/pr-481.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: ConfigurableServiceDiscovery adds WithDefaultParams and WithServiceParams
```

Comment:
Can you expand on this a little bit. For example, what do these do and when would I want to use them as a consumer.